src/rastervision/rv_config.py

```diff
@@ -1,21 +1,95 @@
 import os
 import json
+import tempfile
+from pathlib import Path
 
 from everett.manager import (ConfigManager, ConfigDictEnv, ConfigEnvFileEnv,
                              ConfigIniEnv, ConfigOSEnv)
 
 import rastervision as rv
 from rastervision.utils.files import file_to_str
+from rastervision.filesystem.local_filesystem import make_dir
 
 
 class RVConfig:
     DEFAULT_PROFILE = 'default'
 
+    # We need to use a custom location for the temporary directory so
+    # it will be mirrored on the host file system which is needed for
+    # running in a Docker container with limited space on EC2.
+    DEFAULT_DIR = '/opt/data/tmp/'
+
+    tmp_dir = None
+
+    @staticmethod
+    def get_tmp_dir():
+        tempfile.tempdir = RVConfig.tmp_dir
+        return tempfile.TemporaryDirectory()
```

Comment:
@jamesmcclain This sets the global tempdir. Can you use:

```
tempfile.TemporaryDirectory(dir=RVConfig.tmp_dir)
```
instead?

Reply:
Ah, okay, much appreciated.